docs/setting-up/client/mongodb.md

```diff
@@ -1,20 +1,51 @@
 # MongoDB
 
-## Configuring MongoDB for Monitoring in PMM Query Analytics
+PMM Client collects metrics from [MongoDB][MONGODB] and [Percona Server for MongoDB][PSMDB].
 
-In Query Analytics, you can monitor MongoDB metrics and queries. Run the `pmm-admin add` command to use these monitoring services.
+This page shows you how to set up PMM to monitor a MongoDB database instance. (You should read it completely before starting work.)
```

Comment:
Please read it completely? or just Read it completely. Do we need `()` ?

Reply:
It's a kind of aside, hence in parenthesis. But it's probably more important than that so I will promote it to a box and do the same in the other recently-revised pages (MySQL & PostgreSQL). In which case, yes, it can be more succinct. But it should stay "Read it completely (or in full) before you begin." I.e. Don't just read it completely then do something else. Read it completely before you do anything on this page!